Library/Formula/node.rb

```diff
@@ -55,42 +56,50 @@ def install
     system "./configure", *args
     system "make", "install"
 
-    resource("npm").stage libexec/"npm" if build.with? "npm"
-  end
+    if build.with? "npm"
+      resource("npm").stage libexec/"npm_install"
 
-  def post_install
-    return if build.without? "npm"
+      # make sure npm can find node
+      ENV["PATH"] = "#{Formula["node"].prefix}/bin:#{ENV["PATH"]}"
+
+      # set log level temporarily for npm's `make install`
+      ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
+
+      (libexec/"npm_install").cd { system "./configure", "--prefix=#{libexec}/npm" }
+      (libexec/"npm_install").cd { system "make", "install" }
+
+      # FIXTHIS: Manpages aren't currently linking, which is frustrating.
+      rm_rf libexec/"npm/lib/node_modules/npm/share"
```

Comment:
The usual ` brew failed to post-install because x conflicts ` reference, with a nice list of 50+ manpages listed following :p. It basically happens because we previously installed ` npm ` directly into ` #{HOMEBREW_PREFIX} ` and so ` brew ` isn't great at overwriting those man pages previously symlinked. I speculated with Mike last night this might be because technically speaking, having not *directly* put them there, Brew doesn't think it owns them and can overwrite them. Not 100% on that, but I tried about 4 different methods of linking over them and got the big red postinstall failure each time. Still working on it, but bleh.